src/store/getters.ts

```diff
@@ -196,10 +196,11 @@ export const isRestoration = (state: StateIF): boolean => {
   return (getRequestActionCd(state) === NrRequestActionCodes.RESTORE)
 }
 
-/** True if current flow is XPRO (includes AMALGAMATION, NEW_BUSINESS, and possibly others). */
+/** True if current flow is XPRO (includes AMALGAMATION, NEW_BUSINESS, RESTORATION, and possibly others). */
 export const getIsXproFlow = (state: StateIF): boolean => {
   return (
-    (getRequestActionCd(state) !== NrRequestActionCodes.MOVE) &&
+    ((getRequestActionCd(state) !== NrRequestActionCodes.MOVE) ||
+    (getRequestActionCd(state) !== NrRequestActionCodes.RESTORE)) &&
```

Comment:
Sadly, the comment doesn't match the code.

I think  your intent is to **include** RESTORE, but the code is **excluding** it.

It would be better to include the respective action codes but I think that's risky.

RESTORE is not MOVE so the old code here should work, right?

Reply:
Sev, maybe I'm not understanding correctly. 

So this here, I added an `||` to check if the flow is XPRO (in case the selected business is restorable and XPRO). The reason why I need this is to have access to the select jurisdiction dropdown menu and other fields that are related to XPRO (like MRAS search etc..).

In `search.vue`, there are lots of instances of this getter where we check if we're in XPRO. If this is false, many of the components would not be accessible.

So, I added this to say that if we are in Restoration flow --> there is potential to be in XPRO flow. Hence, this getter would be true. The other logic whether we are in restoration XPRO or not are being handled in `search.vue`.

Reply:
To be honest, I'm kinda confused, but it looks like the old code returned True if this **was not** a MOVE (ie, it was a AML, NEW, REH, etc).

Now it looks it's True if it's not a REH either...? In other words, False for a REH?

Reply:
I think this logic is messed up anyway.

`if (!a || !b)` is the same as `if !(a && b)`, right? But it can't be both a and b.

Another way to look at it is, if it's not a MOVE then it will match everything else (including REH), or if it's not a REH then it will match everything else (including MOVE).

Reply:
The logic currently is:
if (a || b) && c. The old code was if (a && c).

My code only adds on additional case to the expression. If MOVE or REH is true, then the OR expression is true. Meaning that, if we are in MOVE flow or RESTORE flow, we want the XPRO flow to essentially be true.

Now, we could have REH and not XPRO. That is being handled in `search.vue` not here. Here, it's just to add that REH can be in an XPRO flow.

Reply:
The old code was `if (!a && c)`. The new code is `if (!a || !b) && c`.

That first part, I think, is always True. Example: "if not blue or not green".

Reply:
Wait, you are absolutely right. Ohhh, it should be:
`((getRequestActionCd(state) === NrRequestActionCodes.MOVE) || (getRequestActionCd(state) === NrRequestActionCodes.RESTORE))`
This'll only include MOVE and RESTORE. Sev, great great catch. Sorry about that.

Reply:
Wait! The original logic was "everything except MOVE".

I think it would be clearer to say what's included instead of what's excluded.

Also, if it's everything except Move then Restore is already included...?

Reply:
Yes, you're right. Sev, I'm taking a look at it now again and double checking everything.

Reply:
Sev, like you mentioned. I've changed the getter to specify which flows will include the XPRO flow. This way, it's easier to understand and easier to add to. I've also tested the different work flows, looks OK.